apps/emqx_retainer/rebar.config

```diff
@@ -18,7 +18,7 @@
 {profiles,
  [{test,
    [{deps,
-     [{emqx_ct_helpers, {git, "https://github.com/emqx/emqx-ct-helpers", {tag, "1.2.2"}}},
```

Comment:
for this. we can use an override rule from rebar.config in project root.

Reply:
or maybe we don't need this at all for applications in `apps` dir.

Reply:
that works. deleted them from `apps`